psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -214,9 +214,16 @@ def scf_initialize(self):
 
         core.print_out("\n  ==> Pre-Iterations <==\n\n")
 
+        # force SCF_SUBTYPE to AUTO during SCF guess
+        optstash = p4util.OptionsState(["SCF_SUBTYPE"])
+        core.set_local_option("SCF", "SCF_SUBTYPE", "AUTO")
```

Comment:
either the first line needs to be `["SCF", "SCF_SUBTYPE"]` or the second needs to be `set_global_option`

Reply:
Ah, I see. Addressed!